### ISSUE TITLE
Demo: open a content / book

### ISSUE THREAD

**Issue by pmartin** [pmartin/pocketbook-demo#26]
There are two `OpenBook()` and `OpenBook2()` (and some other functions that could be more or less related) functions in `inkview.h`.

It could be interesting trying those, especially to see :
- [x] if they allow reading EPUB files
- [x] and other types, maybe even HTML?
- [x] including - remote - images?
- [x] and the consequences on the system (like: are the opened files necessarily/automatically added to last books read?)
- [x] if everything like paging works out of the box
- [x] how they can be integrated in an application that would list some "items to read"

Basically: if we could re-use a function provided by the system instead of having to code our own reader, it could be incredibly useful ;-)
